exporter.go

```diff
@@ -366,6 +398,35 @@ func (b *Exporter) handleEvent(event Event) {
 			conflictingEventStats.WithLabelValues("counter").Inc()
 		}
 
+	case *FlushEvent:
+		log.Debugf("FlushEvent")
+		for k, v := range b.Sets.store {
+			setevent := b.Sets.events[k]
+
+			b.handleEvent(&GaugeEvent{
+				relative:   false,
+				metricName: setevent.MetricName(),
+				value:      float64(len(v)),
+				labels:     setevent.labels,
+			})
+
+			b.Sets.store[k] = map[float64]bool{}
+		}
+
+	case *SetEvent:
+		store, found := b.Sets.store[metricName]
```

Comment:
what happens if the event has dogstatsd tags attached? I think we need to keep separate sets for each of them.

Reply:
not familiar with dogstatsd, but that makes sense.   Are those stored in Event.Labels()?

Reply:
yes